packages/redirections/lib/Redirections.u.jest.js

```diff
@@ -0,0 +1,553 @@
+    expect(() => validateTarget('https://de.wikipedia.org')).toThrow(expectedErrorMessagee)
+    expect(() => validateTarget('https:/de.wikipedia.org/')).toThrow(expectedErrorMessagee)
+    expect(() => validateTarget('http:de.wikipedia.org/')).toThrow(expectedErrorMessagee)
+    expect(() => validateTarget('http://röpüblique.ch/')).toThrow(expectedErrorMessagee)
+  })
+
+  test('w/ valid URL', () => {
+    const { validateTarget } = Redirections
+
+    expect(validateTarget('/foobar')).toBeUndefined()
+    expect(validateTarget('/foobar/123')).toBeUndefined()
+    expect(validateTarget('/foobar/123?p=456')).toBeUndefined()
+    expect(validateTarget('/foobar/123?p=456#123')).toBeUndefined()
+
+    expect(validateTarget('/foobar/umläute')).toBeUndefined()
+    expect(validateTarget('/foobar/umläute und spaces')).toBeUndefined()
+    expect(validateTarget('/foobar/uml%C3%A4ute%20und%20spaces')).toBeUndefined()
+
+    expect(validateTarget('/~someprofile')).toBeUndefined()
+    expect(validateTarget('/2019/05/31/some-article')).toBeUndefined()
+
+    expect(validateTarget('https://de.wikipedia.org/')).toBeUndefined()
+    expect(validateTarget('https://de.wikipedia.org/wiki/Claus_Kleber')).toBeUndefined()
+    expect(validateTarget('https://de.wikipedia.org/wiki/Claus_Kleber?foo=bar')).toBeUndefined()
+    expect(validateTarget('https://de.wikipedia.org:8080/wiki/Claus_Kleber')).toBeUndefined()
+    expect(validateTarget('http://xn--rpblique-n4a5d.ch/')).toBeUndefined()
+  })
+})
+
+describe('validateStatus()', () => {
+  test('is a function', () => {
+    const { validateStatus } = Redirections
+    expect(typeof validateStatus).toBe('function')
+  })
+
+  test('throw Error if status is neither 301 or 302', () => {
+    const { validateStatus } = Redirections
+    const expectedErrorMessagee = /status .* invalid/
+
+    expect(() => validateStatus(123)).toThrow(expectedErrorMessagee)
+    expect(() => validateStatus('123')).toThrow(expectedErrorMessagee)
+    expect(() => validateStatus('abc')).toThrow(expectedErrorMessagee)
+    expect(() => validateStatus(null)).toThrow(expectedErrorMessagee)
+    expect(() => validateStatus(undefined)).toThrow(expectedErrorMessagee)
+    expect(() => validateStatus(false)).toThrow(expectedErrorMessagee)
+    expect(() => validateStatus({})).toThrow(expectedErrorMessagee)
+    expect(() => validateStatus('301')).toThrow(expectedErrorMessagee)
+    expect(() => validateStatus('302')).toThrow(expectedErrorMessagee)
+  })
+
+  test('w/ status 301 or 302', () => {
+    const { validateStatus } = Redirections
+
+    expect(validateStatus(301)).toBeUndefined()
+    expect(validateStatus(302)).toBeUndefined()
+  })
+})
+
+describe('DEFAULT_ROLES', () => {
+  test('DEFAULT_ROLES contains "admin", "editor"', () => {
+    const expected = ['admin', 'editor']
+    const { DEFAULT_ROLES } = Redirections
+    expect(DEFAULT_ROLES).toEqual(expect.arrayContaining(expected))
+    expect(DEFAULT_ROLES).toHaveLength(2)
+  })
+})
```

Comment:
very nice pg-mocking setup! :+1: 
Altough I think more internals are tested that neccessary. Implementation details like e.g. how many times pgdb.count is called should be hidden from the tests IMHO.